monkey/common/events/stolen_credentials_event.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABC
+from dataclasses import dataclass
+from ipaddress import IPv4Address
+from typing import FrozenSet, Iterable, Union
+from uuid import UUID
+
+from common.credentials import Credentials
+
+
+@dataclass(frozen=True)
+class StolenCredentialsEvent(ABC):
```

Comment:
All events inherit from AbstractEvent.
```suggestion
class StolenCredentialsEvent(AbstractEvent):
```